lms/services/user.py

```diff
@@ -0,0 +1,67 @@
+from lms.models import ApplicationInstance
+from lms.models import LTIUser as User
+from lms.models import display_name as create_display_name
+
+
+class UserService:
+    def __init__(self, db, application_instance_service):
+        self._db = db
+        self._application_instance_service = application_instance_service
+
+    def get(self, oauth_consumer_key, user_id):
+        return (
+            self._db.query(User)
+            .join(ApplicationInstance)
+            .filter(
+                ApplicationInstance.oauth_consumer_key == oauth_consumer_key,
+                User.user_id == user_id,
+            )
+            .one_or_none()
+        )
+
+    def upsert_from_lti(
```

Comment:
`upsert_from_lti` as it takes arguments with the names from the lti launches. I haven't found any need for another simpler method.